app/views/incomeTaxSummaryIncomeSourceDetail.scala.html

```diff
@@ -35,7 +35,7 @@
     <div id="@typeLabel@idx"
          class="subsection flush--top @if(pair.length==1){column-full govuk-grid-column-full}else{column-one-half govuk-grid-column-one-half}">
 
-        <h3 id="@{typeLabel}@{idx}Heading" class="govuk-heading-l heading-large flush break-word-only">
+        <h3 id="@{typeLabel}@{idx}Heading" class="govuk-heading-l heading-large flush break-word">
```

Comment:
btw heading-large is the old unused class that could be removed

Reply:
will do, thanks